src/app/catalog/product/components/product-detail/product-detail.component.ts

```diff
@@ -30,6 +33,14 @@ export class ProductDetailComponent implements OnInit {
     this.store.dispatch(navigate({ url: '/order' }));
   }
 
+  increseProductNumber(): void {
```

Comment:
Can you add a test for this? I think it could be valuable to see how signals are tested.

Reply:
Please recheck Manuel : ]